src/murfey/server/api.py

```diff
@@ -574,6 +574,32 @@ def register_tilt_series(
     db.commit()
 
 
+@router.post("/clients/{client_id}/tilt_series_length")
```

Comment:
I think we can use `"/sessions/{session_id}/tilt_series_length"` to avoid the `session_id` lookup below. On the client you should be able to just use `environment.murfey_session` rather than `environment.client_id`